### DISCUSSION TITLE
Enabling Wiegand

### DISCUSSION THREAD

**Discussion by zizebra** [arendst/Tasmota#21091] [Support request] [answered]
Hi all

Iam using gitpod to compile tasmota with Wiegand enabled. The compile is successfull, I do see the Wiegand option compiling. However when I upload the final upload. I don't find the Wiegand pins for assignment. Can you assist?

**Comment by barbudor** [comment]
Hello
How are you sure that the Wiegand driver is compiled in ?
If the GPIO doesn't appear it's likely you made a mistake
You can use the decode-status tool to confirm

Check your changes and provide details on the exact steps you did to get Wiegand in your build

**Reply by zizebra** [reply, marked as answer]
thank you.  I uncommented the line below in my_user_config.h file.

 #define USE_WIEGAND    

and below is what i noted when i mentioned that i see the option being compiled
![image](https://github.com/arendst/Tasmota/assets/4173129/ad2809ff-ad6b-4af8-b69a-0bf6d5d7d12c)

**Reply by zizebra** [reply]
not sure what was going on. but today, the Wiegend options are now showing on the device. Thank you.
